website/docs/en/config/output.mdx

```diff
@@ -518,16 +614,43 @@ module.exports = {
 };
 ```
 
+## output.hashDigest
+
+- **Type:** `string`
+- **Default:** `'hex'`
+
+The encoding to use when generating the hash. Using `'base64'` for filenames might be problematic since it has the character `/` in its alphabet. Likewise `'latin1'` could contain any character.
+
+## output.hashDigestLength
+
+- **Type:** `number`
+- **Default:** `'20'`
+
+The prefix length of the hash digest to use.
+
+## output.hashFunction
+
+- **Type:** `'md4' | 'xxhash64'`
+- **Default:** `'md4'`
+
+The hashing algorithm to use.
+
+:::tip
+`hashFunction` supports `xxhash64` as a faster algorithm, which will be used as default when `experiments.futureDefaults` is enabled.
```

Comment:
It seems like that Rspack does not supports `futureDefaults`.

Maybe we can enable `xxhash64` by default in v1.0

Reply:
Rspack supports `futureDefaults`. I will include this feature in docs https://github.com/web-infra-dev/rspack/pull/6823/files .